src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.MemOps.cs

```diff
@@ -0,0 +1,537 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if TARGET_AMD64 || TARGET_ARM64 || (TARGET_32BIT && !TARGET_ARM) || TARGET_LOONGARCH64
+#define HAS_CUSTOM_BLOCKS
+#endif
+
+using System.Diagnostics;
+using System.Numerics;
+
+namespace System.Runtime.CompilerServices
+{
+    public static partial class RuntimeHelpers
+    {
+#if NATIVEAOT
+        [System.Runtime.RuntimeExport("RhRuntimeHelpers_MemSet")]
+#endif
+        private static unsafe void MemSet(ref byte dest, byte value, nuint numElements)
```

Comment:
The JIT will convert initblk with zero fill pattern to MemZero. I would think that `initblk` with non-zero fill pattern is extremely rare and something that it is fine to take a small perf hit for to keep things simple by forwarding this call to SpanHelpers.Fill.